integration-test/1491-generic-network-fixes.py

```diff
@@ -0,0 +1,4231 @@
+                'is_in:country': u'Georgia',
+                'name': u'Batumi–Akhaltsikhe Highway',
+                'nat_ref': u'შ 1',
+                'network': u'ge:national',
+                'ref': u'შ 1',
+                'route': u'road',
+                'source': u'openstreetmap.org',
+                'type': u'route',
+            }, ways=[250762262]),
+        )
+
+        self.assert_has_feature(
+            z, x, y, 'roads', {
+                'id': 250762262,
+                'network': u'GE:national',
+            })
+
+    def test_grprovincial22(self):
+        import dsl
+
+        z, x, y = (16, 36383, 24903)
+
+        self.generate_fixtures(
+            dsl.is_in('GR', z, x, y),
+            # https://www.openstreetmap.org/way/503900908
+            dsl.way(503900908, dsl.tile_diagonal(z, x, y), {
+                'highway': u'secondary',
+                'oneway': u'yes',
+                'reg_ref': u'ΕΠ2;ΕΠ16',
+                'source': u'openstreetmap.org',
+            }),
+            dsl.relation(1, {
+                'network': u'gr:provincial:22',
+                'ref': u'2202',
+                'route': u'road',
+                'source': u'openstreetmap.org',
+                'type': u'route',
+            }, ways=[503900908]),
+        )
+
+        self.assert_has_feature(
+            z, x, y, 'roads', {
+                'id': 503900908,
+                'network': u'GR:provincial:22',
+            })
+
+    def test_grprovincial52(self):
+        import dsl
+
+        z, x, y = (16, 37161, 24536)
+
+        self.generate_fixtures(
+            dsl.is_in('GR', z, x, y),
+            # https://www.openstreetmap.org/way/25144610
+            dsl.way(25144610, dsl.tile_diagonal(z, x, y), {
+                'highway': u'secondary',
+                'int_name': u'Drama - Mavrolefki',
+                'lanes': u'2',
+                'name': u'Αμφίπολη - Δράμα',
+                'oneway': u'yes',
+                'reg_ref': u'ΕΠ1',
+                'source': u'openstreetmap.org',
+            }),
+            dsl.relation(1, {
+                'name': u'Δράμα - Μαυρολεύκη - Νέα Μπάφρα',
+                'network': u'gr:provincial:52',
+                'ref': u'5201',
+                'route': u'road',
+                'source': u'openstreetmap.org',
+                'type': u'route',
+            }, ways=[25144610]),
+        )
+
+        self.assert_has_feature(
+            z, x, y, 'roads', {
+                'id': 25144610,
+                'network': u'GR:provincial:52',
+            })
+
+    def test_grprovincial54(self):
+        import dsl
+
+        z, x, y = (16, 36967, 24661)
+
+        self.generate_fixtures(
+            dsl.is_in('GR', z, x, y),
+            # https://www.openstreetmap.org/way/14429261
+            dsl.way(14429261, dsl.tile_diagonal(z, x, y), {
+                'highway': u'secondary',
+                'name': u'Άγιος Βασίλειος - Χορτιάτης',
+                'reg_ref': u'ΕΠ20',
```

Comment:
It this supposed to end up in the network list via your other earlier Greek work?

Reply:
It does (via the relation). The "ΕΠ20" by itself doesn't contain enough information, but the relation has the "full `ref`" of "5420". As with the comment above, it's not clear that this is signed anywhere, so might not be useful to put a shield on.

Reply:
🤷‍♂️ moving on...